community/concepts/deep-then-wide.md

```diff
@@ -1,30 +1,51 @@
 # deep then wide
 
-SourceCred is still a very young project. As we grow it, we have a choice of priorities: 
+SourceCred is still a young project. As we grow it, we have a choice of priorities:
```

Comment:
I think we could take this further and just say "SourceCred is a young project," but this is up to your preference (both are ok).

Reply:
I don't think we're 'very' young anymore. At least toddlers now :)

Reply:
I for one have the soul of an old man! Now where did I leave my cheddar cheese...

Reply:
Updated